app-modules/engagement/src/Filament/Pages/Inbox.php

```diff
@@ -102,6 +104,21 @@ public function table(Table $table): Table
                         return (($record->sender instanceof Student) || ($record->sender instanceof Prospect))
                             ? $record->sender->full_name
                             : null;
+                    })
+                    ->action(function (EngagementResponse $record) {
```

Comment:
I believe you can and probably should use `url()` instead of `action()` here.

That works better for this scenario, is a Filament default functionality, and has other modifications out of the box we can do to it in the future if we wanted.